### PR TITLE
align torch and keras gptq tutorials

### PR DESCRIPTION
**Pull request by irenaby**
## Pull Request Description:
Improve gptq feature tutorials for keras and pytorch.

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mobilenet_gptq.ipynb [hunk 1]
```diff
@@ -129,10 +129,8 @@
     "\n",
     "if not os.path.isdir('imagenet'):\n",
     "    !mkdir imagenet\n",
-    "    !wget https://image-net.org/data/ILSVRC/2012/ILSVRC2012_devkit_t12.tar.gz\n",
-    "    !mv ILSVRC2012_devkit_t12.tar.gz imagenet/\n",
-    "    !wget https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_val.tar\n",
-    "    !mv ILSVRC2012_img_val.tar imagenet/"
+    "    !wget -P imagenet https://image-net.org/data/ILSVRC/2012/ILSVRC2012_devkit_t12.tar.gz\n",
+    "    !wget -P imagenet https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_val.tar"
    ],
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mobilenet_gptq.ipynb:134]: very nice! same textual comments for the pythorch..

**Comment by ofirgo** [comment]
@irenaby 
Please add PR description including the changes you made

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mobilenet_gptq.ipynb [hunk 2]
```diff
@@ -97,21 +116,40 @@
...
+    "        return [cls_to_nid[int(cls)] for cls in f.readlines()]\n",
+    "\n",
+    "if not target_dir.exists():\n",
+    "    labels = extract_labels()\n",
+    "    for lbl in set(labels):\n",
+    "        os.makedirs(target_dir / lbl)\n",
+    "    \n",
+    "    for img_file, lbl in zip(sorted(os.listdir(imgs_dir)), labels):\n",
+    "        shutil.move(imgs_dir / img_file, target_dir / lbl)\n"
    ],
```
[the remark this answers is not in the document]
Comment (irenaby) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mobilenet_gptq.ipynb:140]: Leaving this for the future tutorials alignment task. I think this can be a good alternative, certainly in terms of readability.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mobilenet_gptq.ipynb [hunk 3]
```diff
@@ -125,16 +163,11 @@
...
-    " \n",
-    "In order to decide on the size of the representative dataset, we configure the batch size and the number of calibration iterations.\n",
-    "This gives us the total number of samples that will be used during PTQ (batch_size x n_iter).\n",
-    "In this example we set `batch_size = 50` and `n_iter = 10`, resulting in a total of 500 representative images.\n",
+    "GPTQ is a gradient-based optimization process, which requires representative dataset to perform inference and compute gradients. It is possible to define a separate representative dataset in addition to the one used for the PTQ statistics collection. A possible reason to do so is, for example, to use a larger dataset in the optimization process.\n",
     "\n",
-    "Please ensure that the dataset path has been set correctly."
+    "In this tutorial we use the same representative dataset for both statistics collection and GPTQ. A complete pass through the representative dataset generator constitutes an epoch (batch_size x n_iter samples). In this example we use the same dataloader iterator for all epochs, i.e. different images are used in different epochs."
    ]
```
[the remark this answers is not in the document]
Comment (irenaby) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mobilenet_gptq.ipynb:160]: Most of the explanation already existed, but rephrased and made it shorter.

**Review by Idan-BenAmi — APPROVED** [review]
